tests/suites/refresh/refresh.sh

```diff
@@ -11,7 +11,9 @@ run_refresh_cs() {
 	wait_for "ubuntu" "$(idle_condition "ubuntu")"
 
 	OUT=$(juju refresh ubuntu 2>&1 || true)
-	if echo "${OUT}" | grep -E -vq "Added"; then
+	if echo "${OUT}" | grep -E -q "Added"; then
+	  echo "refresh passed successfully"
```

Comment:
static analysis is complaining, maybe it's because of this line? (maybe indentation?)